docs/source/reference/services.md

```diff
@@ -138,6 +138,14 @@ JUPYTERHUB_OAUTH_SCOPES:   JSON-serialized list of scopes to use for allowing ac
                            (deprecated in 3.0, use JUPYTERHUB_OAUTH_ACCESS_SCOPES).
 JUPYTERHUB_OAUTH_ACCESS_SCOPES: JSON-serialized list of scopes to use for allowing access to the service (new in 3.0).
 JUPYTERHUB_OAUTH_CLIENT_ALLOWED_SCOPES: JSON-serialized list of scopes that can be requested by the oauth client on behalf of users (new in 3.0).
+JUPYTERHUB_PUBLIC_URL: the public URL of the service,
+  e.g. `https://jupyterhub.horse/services/name/`.
```

Comment:
```suggestion
  e.g. `https://jupyterhub.example.org/services/name/`.
```
I think it's best to use official example domains, just in case something ends up on `jupyterhub.horse` in future. If you want a TLD `jupyterhub.example` [also works](https://datatracker.ietf.org/doc/html/rfc2606).

The definition of `JUPYTERHUB_SERVICE_PREFIX` uses `:service-name` instead of `name`, either is fine but I think it should be consistent.